src/OrchardCore.Modules/OrchardCore.ContentFields/Views/ContentPickerField.Edit.cshtml

```diff
@@ -12,6 +12,7 @@
     var searchUrl = Url.RouteUrl(new { area = "OrchardCore.ContentFields", controller = "ContentPickerAdmin", action = "SearchContentItems", part = partName, field = fieldName });
     var vueElementId = $"ContentPicker_{partName}_{fieldName}_{Guid.NewGuid().ToString("n")}";
     var multiple = settings.Multiple.ToString().ToLowerInvariant();
+    var isValidationRequired = settings.Required && !Model.SelectedItems.Any() && !ViewData.ModelState.IsValid;
```

Comment:
This can also be `true` if there is any validation error, from any other field/part, including any other Content Picker Field (keep in mind that you can have multiple such fields on a content type). Better to check for the validation errors with the current prefix. Check from here for inspiration: https://github.com/OrchardCMS/OrchardCore/blob/49aa9efbeff51949ff726e615f1d6d1d4fc63871/src/OrchardCore.Modules/OrchardCore.ContentFields/Drivers/ContentPickerFieldDisplayDriver.cs#L115